apps/web/components/v2/eventtype/EventLimitsTab.tsx

```diff
@@ -31,6 +41,11 @@ export const EventLimitsTab = (props: Pick<EventTypeSetupInfered, "eventType">)
     },
   ];
 
+  useEffect(() => {
+    durationArray.unshift(minimumBookingNoticeType);
+    formMethods.setValue("minimumBookingNotice", displayValue);
```

Comment:
Could we set this as a `defaultValue` on the `minimumBookingNotice` controller? Or wouldn't that work? Would be nice if we could get rid of this effect all together (since the unshift will already be removed if the above comments are fixed).

Reply:
@JeroenReumkens I tried using a `defaultValue` but since react-hook-form makes it so that default values aren't set if they are undefined,  the default value isn't set properly and you see the value as minutes